documentation/airdrop.md

```diff
@@ -0,0 +1,37 @@
+# Lisk Migration Airdrop
+
+## Overview
+
+This page documents the Lisk Migration Airdrop that is implemented in the contract [L2Airdrop.sol](../src/L2/L2Airdrop.sol) and the corresponding deployment script [L2Airdrop.s.sol](../script/L2Airdrop.s.sol). Further, the Merkle tree generator for the maximum airdrop amounts for each Lisk v4 address is implemented
+[here](https://github.com/LiskHQ/lisk-token-claim/tree/main/packages/tree-builder/src/applications/generate-airdrop-merkle-tree).
+
+The goal of the migration airdrop is to reward the existing Lisk community by airdropping LSK tokens proportionally to the migrated LSK amounts if the community members show activity and commitment to the new Lisk L2 via bridging assets, staking tokens and participating in onchain governance.
+
+## Computation of the Airdrop Amount at Token Migration
+
+Depending on the total amount of LSK distributed in the migration airdrop and the LSK token distribution at migration, we compute a value `MIGRATION_AIRDROP_PERCENTAGE` that is the desired percentage of the airdrop in relation to the account balance. The maximum airdrop amount in LSK that a Lisk v4 account owner can claim on the Lisk L2 is then given by:
```

Comment:
Good point from Andreas, it would be better to reformulate in the spirit of:
"We fix the value `MIGRATION_AIRDROP_PERCENTAGE` that is the percentage of the airdrop in relation to the account balance, based on the total amount of LSK distributed in the migration airdrop and the expected airdrop claims."
Also, I don't think we published the value yet

Reply:
Done.

Reply:
I would still remove "Depending on the total amount of LSK distributed in the migration airdrop and the LSK token distribution at migration" in the beginning of the line.

@sergeyshemyakov why do you want to say " based on the total amount of LSK distributed in the migration airdrop"? `MIGRATION_AIRDROP_PERCENTAGE` is not based on the  amount distributed in the airdrop. Or do I misunderstand here something?

Reply:
It's fine for me to provide some rationale. I'm just confused by the given rationale. I don't see how `MIGRATION_AIRDROP_PERCENTAGE` is based on the amount distributed in the airdrop. 

But removing the rationale is also fine for me.

Reply:
Done (I hope it's a correct change).

Reply:
sounds good to me. can be resolved